src/interactions/moderation/anti-raid-nuke.ts

```diff
@@ -2,48 +2,164 @@ import { ApplicationCommandOptionType } from 'discord-api-types/v10';
 	name: 'anti-raid-nuke',
-	description: 'Bans all members that have joined recently, with new accounts',
+	description: 'Handle raids and nukes',
 	options: [
 		{
-			name: 'join',
-			description: 'How old should a member be for the cybernuke to ignore them (server join date)?',
-			type: ApplicationCommandOptionType.String,
-			required: true,
-		},
-		{
-			name: 'age',
-			description: "How old should a member's account be for the cybernuke to ignore them (account age)?",
-			type: ApplicationCommandOptionType.String,
-			required: true,
-		},
-		{
-			name: 'pattern',
-			description: "A pattern the member's username should match (case insensitive, full match)",
-			type: ApplicationCommandOptionType.String,
-		},
-		{
-			name: 'avatar',
-			description: 'The avatar hash the user or member should have (exact match)',
-			type: ApplicationCommandOptionType.String,
+			name: 'manual',
+			description: 'Manually filters and bans members based on various criterias',
+			type: ApplicationCommandOptionType.Subcommand,
+			options: [
+				{
+					name: 'pattern',
+					description: 'The pattern to match against the username',
+					type: ApplicationCommandOptionType.String,
+				},
+				{
+					name: 'confusables',
+					description: 'Whether to use confusables',
+					type: ApplicationCommandOptionType.Integer,
+					choices: [
+						{ name: 'Off', value: 0 },
+						{ name: 'Only Pattern (Pattern will be resistant to confusables, Default)', value: 1 },
+						{ name: 'Only Filter Members (Filter only members with confusables)', value: 2 },
+						{ name: 'Pattern & Filter Members', value: 3 },
+					],
+				},
+				{
+					name: 'insensitive',
+					description: 'Whether the pattern should be case insensitive (default: true)',
+					type: ApplicationCommandOptionType.Boolean,
+				},
+				{
+					name: 'full_match',
+					description: 'Whether the pattern should only match the whole username (default: false)',
+					type: ApplicationCommandOptionType.Boolean,
+				},
+				{
+					name: 'zalgo',
+					description: 'Whether to filter members with zalgo (default: false)',
+					type: ApplicationCommandOptionType.Boolean,
+				},
+				{
+					name: 'join_from',
+					description: 'Minimum join date (Snowflake | Timestamp | Duration | ISO)',
+					type: ApplicationCommandOptionType.String,
+				},
+				{
+					name: 'join_to',
+					description: 'Maximum join date (Snowflake | Timestamp | Duration | ISO)',
+					type: ApplicationCommandOptionType.String,
+				},
+				{
+					name: 'created_from',
+					description: 'Minimum creation date (Snowflake | Timestamp | Duration | ISO)',
+					type: ApplicationCommandOptionType.String,
+				},
+				{
+					name: 'created_to',
+					description: 'Maximum creation date (Snowflake | Timestamp | Duration | ISO)',
+					type: ApplicationCommandOptionType.String,
+				},
+				{
+					name: 'avatar',
+					description: 'The avatar hash (UserId, cdn url, or "none" to match no avatar)',
```

Comment:
Should this be "The avatar (hash, id, ...)"? "hash" does not describe what's listed in the parens

Reply:
I can configure the parser to accept a raw hash, good idea. Also I will clarify the descriptions